axelrod/tests/strategies/test_stalker.py

```diff
@@ -62,3 +62,12 @@ def test_strategy(self):
         self.versus_test(opponent=axl.Cooperator(),
                          expected_actions=actions,
                          match_attributes={"length": 4})
+
+    def test_reset(self):
```

Comment:
The parent `test_reset` method also plays against `Alternator` so I don't believe this is necessary (but it does no harm). Just mentioning it.

Reply:
The intent is to catch a regression in the new `reset` / `__init__` methods or the `set_match_attributes` / `receive_match_attributes` chain by looking for a specific attribute that should be reset. (There may be other such tests and/or it may be in fact useless.)

Reply:
Cool. Thanks. Looks good to me. 👍